interface-definitions/service-console-server.xml.in

```diff
@@ -28,6 +28,14 @@
             </properties>
             <children>
               #include <include/interface/description.xml.i>
+              <leafNode name="name">
```

Comment:
I'd suggest we go with `alias` here. We also might tighten the regex to only allow alpha numeric characters and a few special ones - also 128 is quiet an alias ;). Something like: `[-_a-zA-Z0-9.]{1,128}`